README.md

```diff
@@ -0,0 +1,67 @@
+# Go-SSB Rooms
+
+This repository contains code for a [Secure-Scuttlebutt Room v2](github.com/ssb-ngi-pointer/rooms2/) server writen in Go.
+
+It not only includes the secret-handshake+boxstream setup and muxrpc handlers for tunneling connections but also a fully embedded http/html interface for administering the room.
+
+## Features
+
+* [x] Rooms v1 (`tunnel.connect`, `tunnel.endpoints`, etc.)
+* [ ] Sign-in with SSB
+* [x] Simple whitelisting
```

Comment:
I know this was probably not an intentional use of the word whitelisting, but "allowlist"/"blocklist" are considered less racist words for whitelist/blacklist. :)